### ISSUE TITLE
请问为什么在训练和merge完lora推理测试的时候log没有打印出custom_val_dataset中的问题和回答

### ISSUE THREAD

**Issue by wintercat1994** [modelscope/ms-swift#1322]
好像还没运行就end了
感谢回复，我的运行命令和log如下：

CUDA_VISIBLE_DEVICES=0 swift export     --ckpt_dir /wangliyao/swift/output/deepseek-vl-1_3b-chat/v4-20240708-172229/checkpoint-2-merged --load_dataset_config true --custom_val_dataset_path toy_dataset.json

run sh: `python /root/miniconda3/envs/sd/lib/python3.10/site-packages/swift/cli/export.py --ckpt_dir /wangliyao/swift/output/deepseek-vl-1_3b-chat/v4-20240708-172229/checkpoint-2-merged --load_dataset_config true --custom_val_dataset_path toy_dataset.json`
[INFO:swift] Successfully registered `/root/miniconda3/envs/sd/lib/python3.10/site-packages/swift/llm/data/dataset_info.json`
[INFO:swift] Start time of running main: 2024-07-08 18:25:36.670228
[INFO:swift] Using val_dataset, ignoring dataset_test_ratio
[INFO:swift] ckpt_dir: /wangliyao/swift/output/deepseek-vl-1_3b-chat/v4-20240708-172229/checkpoint-2-merged
[INFO:swift] Setting model_info['revision']: master
[INFO:swift] Setting self.eval_human: False
[INFO:swift] Setting overwrite_generation_config: True
[INFO:swift] args: ExportArguments(model_type='deepseek-vl-1_3b-chat', model_id_or_path='deepseek-ai/deepseek-vl-1.3b-chat', model_revision='master', sft_type='full', template_type='deepseek-vl', infer_backend='pt', ckpt_dir='/wangliyao/swift/output/deepseek-vl-1_3b-chat/v4-20240708-172229/checkpoint-2-merged', load_args_from_ckpt_dir=True, load_dataset_config=True, eval_human=False, seed=42, dtype='bf16', dataset=['toy_dataset.json'], val_dataset=['toy_dataset.json'], dataset_seed=42, dataset_test_ratio=0.0, show_dataset_sample=10, save_result=True, system='You are a helpful language and vision assistant. You are able to understand the visual content that the user provides, and assist the user with a variety of tasks using natural language.', tools_prompt='react_en', max_length=None, truncation_strategy='delete', check_dataset_strategy='none', model_name=[None, None], model_author=[None, None], quant_method='awq', quantization_bit=0, hqq_axis=0, hqq_dynamic_config_path=None, bnb_4bit_comp_dtype='bf16', bnb_4bit_quant_type='nf4', bnb_4bit_use_double_quant=True, bnb_4bit_quant_storage=None, max_new_tokens=2048, do_sample=True, temperature=0.3, top_k=20, top_p=0.7, repetition_penalty=1.0, num_beams=1, stop_words=None, rope_scaling=None, use_flash_attn=None, ignore_args_error=False, stream=True, merge_lora=False, merge_device_map='auto', save_safetensors=True, overwrite_generation_config=True, verbose=None, local_repo_path=None, custom_register_path=None, custom_dataset_info=None, device_map_config_path=None, gpu_memory_utilization=0.9, tensor_parallel_size=1, max_model_len=None, disable_custom_all_reduce=True, enforce_eager=False, vllm_enable_lora=False, vllm_max_lora_rank=16, lora_modules=[], image_input_shape=None, image_feature_size=None, self_cognition_sample=0, train_dataset_sample=-1, val_dataset_sample=None, safe_serialization=None, model_cache_dir=None, merge_lora_and_save=None, custom_train_dataset_path=[], custom_val_dataset_path=['toy_dataset.json'], vllm_lora_modules=None, to_peft_format=False, quant_bits=0, quant_n_samples=256, quant_seqlen=2048, quant_device_map='cpu', quant_output_dir=None, push_to_hub=False, hub_model_id=None, hub_token=None, hub_private_repo=False, commit_message='update files')
[INFO:swift] Global seed set to 42
[INFO:swift] End time of running main: 2024-07-08 18:25:36.710787
